docs/source/loading.mdx

```diff
@@ -68,6 +68,13 @@ The `split` parameter can also map a data file to a specific split:
 >>> c4_validation = load_dataset("allenai/c4", data_files=data_files, split="validation")
 ```
 
+Datasets hosted on the Hub can also be loaded from a relative path through the `data_dir` or `data_files` parameters. The dataset's relative path is resolved to its URL in the repository where the loading script is stored.
```

Comment:
Thanks !

I think I would simply mention `data_dir` in the previosu paragraph:
```
You can also load a specific subset of the files with the `data_files` or `data_dir` parameter.
```

This way it is more explicit that they work the same way.

The example is since as well :)

Reply:
I agree this sentence here may lead to confusion:
- the sentence mentions a repository with a loading script
- but this section is about Hub datasets without loading script